File: public/phase-2-20160620022203/JAVA/two-piles-of-balls.java
// Name : Two Piles of Balls
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n : number of balls in a pile
		// a : 
		int index;
		Scanner in = new Scanner(System.in);
		int n = Integer.parseInt(in.nextLine());
		int[] a = new int[2n];
		String[] a_elements = (in.nextLine()).split(" ");
		for(index=0;index<2n;index++)
			a[index] = Integer.parseInt(a_elements[index]);


		// write your code here
		// store your results in `result`,`b`

		// output
		// Dummy Data
		int result=1;
		intb[] = new int[] {2,1};

		System.out.println(result);
		System.out.println(join_string(Arrays.toString(b)));

	}
	public static String join_string(String inp) {
		return inp.replace("[","").replace("]","").replace(",","");
	}
}

File: public/phase-2-20160620022203/JAVASCRIPT/two-piles-of-balls.js
// Name : Two Piles of Balls
// input:
// n : number of balls in a pile
// a : 
var index,result,b;
var n = parseInt(readline(), 10);
var a = [];
var a_elements = readline().split(' ');
for(index=0;index<(2*n);index++)
	a.push(parseInt(a_elements[index], 10));


// write your code here
// store your results in `result`,`b`

// Dummy Data
result = 1;
b = [2,1];


// output
print(result);
print(b.join(" "))
